src/libraries/System.DirectoryServices/src/System/DirectoryServices/ActiveDirectory/ADAMInstance.cs

```diff
@@ -695,7 +695,7 @@ public override string SiteName
                     DirectoryEntry siteEntry = DirectoryEntryManager.GetDirectoryEntry(context, SiteObjectName);
                     try
                     {
-                        cachedSiteName = (string)PropertyManager.GetPropertyValue(context, siteEntry, PropertyManager.Cn);
+                        cachedSiteName = (string)PropertyManager.GetPropertyValue(context, siteEntry, PropertyManager.Cn)!;
```

Comment:
Same question for these.

Reply:
Here because code was expecting SiteName to be non-null so I assume this must be returning non-null. This is coming from the network so I based many annotations on how they were used as I don't have any other information